mmv1/third_party/terraform/tests/resource_compute_firewall_policy_association_test.go

```diff
@@ -26,6 +26,8 @@ func TestAccComputeFirewallPolicyAssociation_basic(t *testing.T) {
 				ResourceName:      "google_compute_firewall_policy_association.default",
 				ImportState:       true,
 				ImportStateVerify: true,
+				// Referencing using ID causes import to fail
```

Comment:
Just want to make sure I'm on the same page here- there's a mismatch of values in state because the import is stateless, and this wouldn't happen is ISV checked using a DSF?

Reply:
Yup, this field has a `compareSelfLinkOrResourceName` DSF which would fix this if it was run during ISV.

Because there is no prior state the DCL will return the short name as a best-guess of what we want, which gets compared against the `id` of the parent policy without the DSF and causes a diff